### PR TITLE
Unit-tests

### PR DESCRIPTION
**Pull request by Rusty-Beard**

### CODE REVIEW

File: py/ctl/tasks.py [hunk 1]
```diff
@@ -18,7 +18,6 @@ def task_view(id):
     task_data = dao.tasks.load_one(id_clean)
     if task_data is None:
         flask.abort(404)
-        return
```
Comment (RodionGork) [inline py/ctl/tasks.py]: Is this intended removal or some occasional change? :)

Comment (Rusty-Beard) [reply py/ctl/tasks.py]: individual commit for this) yes, it's unreachable return.

File: py/tests/test_tasks.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import unittest
+from unittest.mock import patch
+import ctl.tasks
+from app import app
+
+
+class TasksViewTestCase(unittest.TestCase):
+    @patch('ctl.tasks.flask')
+    @patch('dao.utils.mysql')
+    def test_connect_to_database(self, mysql_connector, flask):
```
Comment (Rusty-Beard) [inline py/tests/test_tasks.py]: we should rename this function when merging. it has been given when i were trying to understand what to do with the mySQL.

File: py/ctl/tasks.py (hunk 1 above)
Comment (RodionGork) [reply py/ctl/tasks.py]: A-a-a, thanks! This is my lack of experience with flask. I failed to check documentation and see that `abort` raises exception... Thanks a lot!

File: py/tests/test_tasks.py (hunk 2 above)
Comment (RodionGork) [reply py/tests/test_tasks.py]: Ah, ok, thanks - I'll take care of this... You are right, this is a tricky question, whether we should mock mysql or dao layer itself... Anyway it wouldn't be hard for me to change later after you showed the way :+1:

File: py/ctl/tasks.py (hunk 1 above)
Comment (Rusty-Beard) [reply py/ctl/tasks.py]: Due to that return in my tests everything was okay haha. will try to correct

Comment (RodionGork) [reply py/ctl/tasks.py]: Ah, thanks, I thought I can do it myself in the morning when I'm less sleepy :)

**Pull request merged**